tox.ini

```diff
@@ -217,3 +218,13 @@ commands =
     devpi-server \
     --indent 2 \
     -o testing/devpi-server.lock
+
+[testenv:build-cache-image]
```

Comment:
Automating this (and re-creation / push of the base image as needed well) is tracked in #2233.